src/status_im2/contexts/wallet/receive/view.cljs

```diff
@@ -0,0 +1,109 @@
+(ns status-im2.contexts.wallet.receive.view
+  (:require
+    [quo.core :as quo]
+    [react-native.blur :as blur]
+    [react-native.core :as rn]
+    [react-native.platform :as platform]
+    [react-native.safe-area :as safe-area]
+    [react-native.share :as share]
+    [reagent.core :as reagent]
+    [status-im2.constants :as constants]
+    [status-im2.contexts.wallet.common.sheets.network-preferences.view :as network-preferences]
+    [status-im2.contexts.wallet.receive.style :as style]
+    [utils.i18n :as i18n]
+    [utils.image-server :as image-server]
+    [utils.re-frame :as rf]))
+
+(defn- get-network-short-name-url
```

Comment:
Namespace `status-im2.common.qr-codes.view` already contains this function, so it can be reused